src/NLog/Targets/DefaultJsonSerializer.cs

```diff
@@ -432,26 +432,6 @@ private void SerializeCollectionObject(IEnumerable value, StringBuilder destinat
             destination.Append(']');
         }
 
-        private bool SerializeTypeCodeValue(object value, TypeCode objTypeCode, StringBuilder destination, JsonSerializeOptions options, SingleItemOptimizedHashSet<object> objectsInPath, int depth)
```

Comment:
this is not needed anymore? Please do describe these changes in the PR title to make the review less of a surprise - thanks!

Reply:
I will update it for now

Reply:
Thx